Makefile

```diff
@@ -42,7 +42,8 @@ WHITELISTED_THIRDPARTY_ASSEMBLIES = ICSharpCode.SharpZipLib.dll FuzzyLogicLibrar
 
 # These are shipped in our custom minimal mono runtime and also available in the full system-installed .NET/mono stack
 # This list *must* be kept in sync with the files packaged by the AppImageSupport and OpenRALauncherOSX repositories
-WHITELISTED_CORE_ASSEMBLIES = mscorlib.dll System.dll System.Configuration.dll System.Core.dll System.Numerics.dll System.Security.dll System.Xml.dll Mono.Security.dll netstandard.dll
+WHITELISTED_CORE_ASSEMBLIES_MONO = mscorlib.dll System.dll System.Configuration.dll System.Core.dll System.Numerics.dll System.Security.dll System.Xml.dll Mono.Security.dll netstandard.dll Microsoft.Win32.Registry.dll System.Security.AccessControl.dll System.Security.Principal.Windows.dll System.Security.AccessControl.dll System.Xml.Linq.dll System.Runtime.Serialization.dll System.Security.Principal.Windows.dll
+WHITELISTED_CORE_ASSEMBLIES_NETCORE = System.Private.CoreLib.dll System.Runtime.dll System.Collections.dll System.Runtime.Extensions.dll System.IO.FileSystem.dll System.Linq.dll System.Console.dll System.Threading.dll System.Text.RegularExpressions.dll System.Diagnostics.Process.dll System.ComponentModel.Primitives.dll System.Text.Encoding.Extensions.dll Microsoft.Win32.Primitives.dll System.Memory.dll System.Runtime.InteropServices.dll System.Collections.Concurrent.dll System.ComponentModel.TypeConverter.dll System.Runtime.Numerics.dll System.Buffers.dll System.Threading.Thread.dll System.Security.Cryptography.Algorithms.dll System.Security.Cryptography.Primitives.dll System.Threading.Tasks.dll System.Net.Primitives.dll System.Linq.Expressions.dll System.Runtime.Serialization.Formatters.dll System.Diagnostics.TraceSource.dll System.ObjectModel.dll System.Security.Claims.dll System.Security.Principal.dll System.Collections.NonGeneric.dll
```

Comment:
We needed this for mono because we were packaging a deliberately stripped down runtime missing many standard assemblies so needed to make sure that we weren't referencing anything that we didn't ship.

This is now managed for us by `dotnet`, so I think we can skip this and only run the assembly checks for mono?